crates/turbopack-node/src/evaluate.rs

```diff
@@ -30,7 +33,7 @@ pub enum JavaScriptValue {
 
 async fn eval_js_operation(
     operation: &mut NodeJsOperation,
-    content: EvalJavaScriptOutgoingMessage,
+    content: EvalJavaScriptOutgoingMessage<'_>,
 ) -> Result<String> {
     operation.send(content).await?;
     match operation.recv().await? {
```

Comment:
This `EvalJavaScriptIncomingMessage::Error` case needs work, because it's leaking a Rust stack trace into our otherwise good error message:

```
Failed to load external module tailwindcss: Error: Cannot find module 'tailwindcss'
Require stack:
- /Users/jridgewell/src/vercel/turbo/example/.next/build/postcss/chunks/[project-with-next]_tailwind.config_52e1ea.js
- /Users/jridgewell/src/vercel/turbo/example/.next/build/postcss/transform.js
warning - [parse]
  ... [2 paths] are hidden, run with --show-all to show them
event - updated in 11s
[500] error: TypeError: Cannot read properties of undefined (reading 'default')
  at <unknown> (/Users/jridgewell/src/vercel/turbo/example/.next/build/postcss/chunks/[project-with-next]_postcss.config.js_transform.js:46:31)
  at <anonymous>
  at Module.transform (/Users/jridgewell/src/vercel/turbo/example/.next/build/postcss/chunks/[project-with-next]_postcss.config.js_transform.js:37:35)
  at <unknown> (/Users/jridgewell/src/vercel/turbo/example/.next/build/postcss/chunks/[project-with-next]_tailwind.config_52e1ea.js:23:197)
  at Module.run (/Users/jridgewell/src/vercel/turbo/example/.next/build/postcss/chunks/[turbopack-node]_ipc_evaluate.ts.js:19:45)
  at process.processTicksAndRejections (node:internal/process/task_queues:95:5)


Stack backtrace:
   0: std::backtrace_rs::backtrace::libunwind::trace
             at /rustc/215e3cd218b83b8a3152d84d92f17109253c25e1/library/std/src/../../backtrace/src/backtrace/libunwind.rs:93:5
   1: std::backtrace_rs::backtrace::trace_unsynchronized
             at /rustc/215e3cd218b83b8a3152d84d92f17109253c25e1/library/std/src/../../backtrace/src/backtrace/mod.rs:66:5
   2: std::backtrace::Backtrace::create
             at /rustc/215e3cd218b83b8a3152d84d92f17109253c25e1/library/std/src/backtrace.rs:332:13
   3: anyhow::error::<impl core::convert::From<E> for anyhow::Error>::from
             at /Users/jridgewell/.cargo/registry/src/github.com-1ecc6299db9ec823/anyhow-1.0.66/src/error.rs:547:25
   4: <T as core::convert::Into<U>>::into
             at /rustc/215e3cd218b83b8a3152d84d92f17109253c25e1/library/core/src/convert/mod.rs:726:9
   5: turbo_tasks_memory::output::Output::read_untracked
             at ./crates/turbo-tasks-memory/src/output.rs:55:46
   6: <turbo_tasks_memory::memory_backend::MemoryBackend as turbo_tasks::backend::Backend>::try_read_task_output_untracked::{{closure}}
             at ./crates/turbo-tasks-memory/src/memory_backend.rs:283:22
```